### PR TITLE
Fix targeted apply and add basic example

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
## Description

Since `aws-ia/vpc/aws` module has been bumped to `v2.1.0` this deploy does not work, as the `vpc` module `private_subnet_attributes_by_az` output returns `private/<az_name>` instead of just `<az_name>` .

This PR locks down the `aws-ia/vpc/aws` module version to alleviate this issue, and in the future we'll work on rewriting the scripts to support higher versions of `aws-ia/vpc/aws` module. Or if the `private/` prefix gets removed, we'll update this repo accordingly. 

Removed the need to run `-target` for VPC module, before running `terraform apply`

Added a basic example for deployment

### CODE REVIEW

File: .header.md [hunk 1]
```diff
@@ -55,13 +55,19 @@ This deployment uses `ubuntu-focal-20.04-amd64-server` AWS AMI. It will **not**
...
+Example:
 ```shell
-terraform taint module.instances[\"us-west-2a\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2b\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2c\"].aws_instance.polygon_edge_instance
-terraform taint module.instances[\"us-west-2d\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"0\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"1\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"2\"].aws_instance.polygon_edge_instance
+terraform taint module.instances[\"3\"].aws_instance.polygon_edge_instance
```
Comment (drewmullen) [inline .header.md:65]:
```suggestion
terraform taint module.instances[0].aws_instance.polygon_edge_instance
terraform taint module.instances[1].aws_instance.polygon_edge_instance
terraform taint module.instances[2].aws_instance.polygon_edge_instance
terraform taint module.instances[3].aws_instance.polygon_edge_instance
```

File: examples/basic/main.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module "polygon-edge" {
+  source  = "aws-ia/polygon-technology-edge/aws"
+  version = "0.0.1"
```
Comment (drewmullen) [inline examples/basic/main.tf:3]: so we dont have to update the example every new feature :D
```suggestion
  version = ">= 0.0.1"
```

**Review by drewmullen — CHANGES_REQUESTED** [review]
Really great work. This will be a _much_ better user experience than the targeted apply!

2 small changes then we're good to go!

File: .header.md (hunk 1 above)
Comment (ZeljkoBenovic) [reply .header.md]: Great catch. Key used to be of string type, and I forgot to delete `" "`. Fixed in cb7044370229b899ed7d9cd4831705b9108f30e3

File: examples/basic/main.tf (hunk 2 above)
Comment (ZeljkoBenovic) [reply examples/basic/main.tf]: Fixed in f190b110c8f5e63a56ae941ddfefec24e3d48fb1

**Comment by drewmullen** [comment]
Nice work!

**Pull request merged**